release/e2e.py

```diff
@@ -1280,19 +1334,27 @@ def _run(logger):
             if smoke_test:
                 cmd_to_run += " --smoke-test"
 
-            scd_id, status_code = run_session_command(
+            scd_id, result = run_session_command(
                 sdk=sdk,
                 session_id=session_id,
                 cmd_to_run=cmd_to_run,
-                stop_event=stop_event,
                 result_queue=result_queue,
                 env_vars=env_vars,
-                state_str="CMD_RUN",
-                kick_off_only=kick_off_only)
+                state_str="CMD_RUN")
 
             if not kick_off_only:
+                _, runtime = wait_for_session_command_to_complete(
```

Comment:
Separating wait & run will fix the bug where the last logs are not printed upon timeout. The root cause of the bug is that scd_id is only returned after we finished waiting for the command, so that we couldn't obtain the scd_id to get last logs if we have timeout error (because scd_id wasn't returned)